content/en/docs/tasks/administer-cluster/kubelet-in-userns.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Running Kubernetes node components as a non-root user
+content_type: task
+min-kubernetes-server-version: 1.22
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.22" state="alpha" >}}
+
+This document describes how to run Kubernetes node components such as kubelet, CRI, OCI, and CNI
+without root privileges, by using a {{< glossary_tooltip text="user namespace" term_id="userns" >}}.
+
+This technique is also known as _rootless mode_.
+
+{{< note >}}
+This document describes how to run Kubernetes node components (and hence pods) a non-root user.
+
+If you are just looking for how to run a pod as a non-root user, see [SecurityContext](/docs/tasks/configure-pod-container/security-context/).
+{{< /note >}}
+
+## {{% heading "prerequisites" %}}
+
+* [cgroup v2](https://rootlesscontaine.rs/getting-started/common/cgroup2/)
+* [systemd with user session](https://rootlesscontaine.rs/getting-started/common/login/)
+* [several sysctl values, depending on host Linux distribution](https://rootlesscontaine.rs/getting-started/common/sysctl/)
+* [`/etc/subuid` and `/etc/subgid`](https://rootlesscontaine.rs/getting-started/common/subuid/)
+
+* [`KubeletInUserNamespace` feature gate](/docs/reference/command-line-tools-reference/feature-gates/)
```

Comment:
We should expand this into a paragraph explaining what the reader should do. Also, hyperlink the text “feature gate” to `/docs/reference/command-line-tools-reference/feature-gates/`.